src/Generator/PdfOptionsGenerator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Generator;
+
+use Symfony\Component\Config\FileLocatorInterface;
+
+final class PdfOptionsGenerator implements PdfOptionsGeneratorInterface
```

Comment:
I'm wondering how could be extract common logic from this and invoicing plugin

Reply:
To snappy maybe?! 😄 
Or yet another package in Sylius Stack... 🤷‍♂️